client/app/components/Input/InputSelect.jsx

```diff
@@ -33,14 +34,22 @@ export class InputSelect extends React.Component<Props, State> {
   };
 
   render() {
-    const { id, name, options } = this.props;
+    const {
+      id, name, options, ariaLabel,
+    } = this.props;
     const { value } = this.state;
     return (
       <div className={css.select}>
         <div className={css.selectIcon}>
           <FontAwesomeIcon icon={faCaretDown} />
         </div>
-        <select id={id} name={name} value={value} onChange={this.toggleValue}>
+        <select
+          id={id}
+          name={name}
+          aria-label={ariaLabel}
```

Comment:
Let's do `aria-label={label || ariaLabel}` and assume a select input field with a label can also be used as the aria-label.
And then add label to line 38.